languages/python/oso/polar/ffi.py

```diff
@@ -24,21 +24,23 @@ def load_str(self, string, filename):
         string = to_c_str(string)
         filename = to_c_str(str(filename)) if filename else ffi.NULL
         check_result(lib.polar_load(self.ptr, string, filename))
+        process_messages()
 
     def new_query_from_str(self, query_str):
-        return Query(
-            check_result(lib.polar_new_query(self.ptr, to_c_str(query_str), 0))
-        )
+        query = check_result(lib.polar_new_query(self.ptr, to_c_str(query_str), 0))
```

Comment:
Same things as with Java?

Reply:
Undid the java one lol. Defined which calls this has to come after in the README and then only put the calls in those methods. It's less than check_result uses so can't call it from in there anymore.